program/rust/src/processor/set_min_pub.rs

```diff
@@ -51,7 +51,6 @@ pub fn set_min_pub(
         &cmd.header,
     )?;
 
-
     let mut price_account_data = load_checked::<PriceAccount>(price_account, cmd.header.version)?;
```

Comment:
slightly annoyed at the formatting changes

Reply:
Not sure how that happened, must be rustfmt. Fixed.